earth2mip/networks/pangu.py

```diff
@@ -194,6 +194,10 @@ def grid(self):
     def n_history(self):
         return 0
 
+    @property
+    def device(self) -> torch.device:
+        return "cuda"
```

Comment:
Sorry, I know I told you this, but we can make this just a little more robust with
```
from modulus.distributed.manager import DistributedManager
```


```
@property
    def device(self) -> torch.device:
        return DistributedManager().device
```